pkg/didcomm/crypto/jwe/authcrypt/authcrypt_test.go

```diff
@@ -182,4 +440,16 @@ func TestRefEncrypt(t *testing.T) {
 	require.NoError(t, err)
 	t.Logf("Reference JWE Ummarshalled: %s", refPldUnmarshalled)
 	t.Logf("Encrypted JWE Ummarshalled: %s", encryptedPldUmarshalled)
+
+	// try to decrypt the encrypted payload
+	recipientPrivK := &[32]byte{}
+	copy(recipientPrivK[:], recipientPriv)
+	dec, err := crypter.Decrypt(pld, recipientPrivK)
+	require.NoError(t, err)
+	require.NotEmpty(t, dec)
+
+	// TODO fix try to decrypt the reference payload
```

Comment:
How do we know this PR is working?

Reply:
working on php interoperable decryption/encryption..

Reply:
Why is this line commented out then?